test/org/opensolaris/opengrok/configuration/ProjectTest.java

```diff
@@ -134,4 +132,52 @@ public void testGetProjectDescriptions() {
         assertFalse(descs.contains("foobar"));
         assertEquals(2, descs.size());
     }
+
+    /**
+     * Insert the value from configuration.
+     */
+    @Test
+    public void testMergeProjects1() {
+        Configuration cfg = new Configuration();
+        cfg.setTabSize(8);
```

Comment:
using some crazy number as a value might be more robust (in case default value gets set to 8)

Reply:
fixed